ixofoundation/ixo-world
# chore(bug): fix animation bug on safari

## Scope
Fix animation bug that happens on Safari

## Work done
Changed keyframe opacity from 1 to 0 at 0%

## Steps to test
Run on Safari and let animation cycle for 3 rounds. Confirm no animation glitch occurs.

## Screenshot
![Uploading Screenshot 2020-05-14 at 14.03.41.png…]()
